doc/api/esmvalcore.api.recipe.rst

```diff
@@ -59,7 +59,12 @@ Sessions can also be explicitly specified.
     >>> output = recipe.run(session)
     <log messages>
 
-:py:meth:`esmvalcore.experimental.Recipe.run` returns an object that contains the locations of the data and figures (not implemented yet).
+:py:meth:`esmvalcore.experimental.Recipe.run` returns an dictionary of objects that can be used to inspect
+the output of the recipe. The output is an instance of :py:class:`esmvalcore.experimental.recipe_output.ImageItem` or
+:py:class:`esmvalcore.experimental.recipe_output.ImageItem` depending on its type.
+
+For working with recipe output, see: ref:`_api_recipe_output`.
```

Comment:
```suggestion
For working with recipe output, see: :ref:`_api_recipe_output`.
```

Reply:
Apparently the label should not include the underscore... 🙄

    :ref:`api_recipe_output`